types/data.d.ts

```diff
@@ -2,10 +2,11 @@ import PartyTracker from '../resources/party.js';
 import { Lang } from './global';
 import { Job, Role } from './job';
 
-export interface Option {
+export interface BaseOptions {
```

Comment:
I also changed this to `BaseOptions` with the idea that we'd have `RaidbossOptions extends BaseOptions` later.